src/data/set/intervals/basic.lean

```diff
@@ -328,6 +328,14 @@ lemma Ici_inter_Iio : Ici a ∩ Iio b = Ico a b := rfl
 lemma Ioi_inter_Iic : Ioi a ∩ Iic b = Ioc a b := rfl
 lemma Ioi_inter_Iio : Ioi a ∩ Iio b = Ioo a b := rfl
 
+lemma Icc_of_Ioo (x ∈ Ioo a b) : (x ∈ Icc a b) := Ioo_subset_Icc_self H
```

Comment:
Is it bad form to rely on the generated hypothesis name? I'd be inclined to prefer
```suggestion
lemma Icc_of_Ioo : x ∈ Ioo a b → x ∈ Icc a b := Ioo_subset_Icc_self
```
or
```suggestion
lemma Icc_of_Ioo (h : x ∈ Ioo a b) : x ∈ Icc a b := Ioo_subset_Icc_self h
```

Reply:
I think that's a good point. If other maintainers agree (and see the value in adding these lemmas entirely) I'll go ahead and make those changes.

Reply:
Please rename lemmas like this:
```suggestion
lemma mem_Icc_of_Ioo (h : x ∈ Ioo a b) : x ∈ Icc a b := Ioo_subset_Icc_self h
```
Then we'll merge it.